src/index.scss

```diff
@@ -1,9 +1,19 @@
-$feature-flags: (
```

Comment:
the `feature-flags` class should not be deleted.

Reply:
Thanks for the feedback Kelly. Updated to re-add the feature-flags class.